docs/DeveloperGuide.md

```diff
@@ -410,6 +410,25 @@ otherwise)
 
    Use case ends.
 
+**Use case: UC9 - Delete all items in a specific location**
+
+**MSS**
+
+1. User requests to delete all items in that specific location.
```

Comment:
I think can it would be better to phrase it "User requests to delete all items in a specific location".